src/go/k8s/README.md

```diff
@@ -24,34 +27,30 @@ Optionaly to run operator locally:
 
 #### Local installation
 
-First clone the repo
-
-```
-git clone https://github.com/vectorizedio/redpanda.git
-```
-
 Create local Kubernetes cluster using KIND
 
 ```
 export KUBECONFIG=your/path/to/kubeconfig.yaml
 kind create cluster --config kind.yaml
 ```
 
-You can simply deploy the latest version stored in GCR
+You can simply deploy the Redpanda operator by running the following command
 
 ```
-make deploy
+kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/default
 ```
 
-Or you can build and load the container to Kubernetes cluster
+Install sample RedpandaCluster custom resource
 
 ```
-# Only if you want to build container from source!
-make docker-build deploy-to-kind
+kubectl apply -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/redpanda_v1alpha1_cluster.yaml
 ```
 
-Install sample RedpandaCluster custom resource
+#### Clean up
+
+To remove all resources even the running Redpanda cluster
+please run the following command:
 
 ```
-kubectl apply -f config/samples/redpanda_v1alpha1_cluster.yaml
+kubectl delete -k config/default
```

Comment:
This needs to be `kubectl delete -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/default` since we operate on the premise that the user hasn't cloned the repo.

Reply:
Thank you! fixed.